docs/modules/operation/pages/deep-dive/topology/enlinkd/introduction.adoc

```diff
@@ -57,6 +57,15 @@ CDP, LLDP, OSPF and ISIS collectors provide the required information about links
 Bridge Collectors are able to collect Bridge Forwarding Table and IpNetToMedia Table on devices and to provide effective links a further step is required.
 The Enlinkd-Bridge-Discovery process attempts to discover bridge domain links with the data coming from all collected Bridge Forwarding Tables.
 The Topology-Updaters correlate all collected topology data to generate a global topology layout of your network.
+For each supported protocol is created a collection group with an assigned priority.
+The collection group is scheduled according with the rescan interval defined for the related protocol.
+The run of collection group will send to a Priority Executor protocol specific Snmp Collection and storage.
+First will be collecter the data whose value of priority is lower.
+Further a priority Map is ised to assign to a specific node a priority.
```

Comment:
```suggestion
Further, collection uses a priority map to assign a priority to a specific node.
```
Where is the priority map? How does the user define what priority specific nodes get? We should include a link to this information if it is somewhere else in the docs.

Reply:
I commit the suggested changes, but maybe you can find a better way for explaining what I have done.
Snmp Collection is done first using a protocol criteria so CDP data collection is always performed before LLDP data collection because cdp-priority=1000 and lldp-priority=2000.
Also a criteria is used to set up data collection priority that is the sum of protocol-.priority and node-priority.

Reply:
Thanks. I'll look at the updates and see if we need to change anything.